modules/fedimint-lnv2-client/src/lib.rs

```diff
@@ -984,6 +986,88 @@ impl LightningClientModule {
 
         Ok(state)
     }
+
+    /// Generate an lnurl for the client.
+    pub async fn generate_lnurl(
+        &self,
+        recurringd: Option<SafeUrl>,
+        gateway: Option<SafeUrl>,
+    ) -> Result<String, RegisterLnurlError> {
+        let recurringd = match recurringd {
+            Some(recurringd) => recurringd,
+            None => SafeUrl::from_str("https://recurringdv2.fedimint.org").unwrap(),
```

Comment:
IMO this is the wrong layer to put a hardcoded constant. At the very least it needs to be documented, otherwise downstream consumers will not know what the expected behavior of passing `None` for `recurringd`